src/modules/plugins/plugins.service.ts

```diff
@@ -859,8 +859,11 @@ export class PluginsService {
       throw new NotFoundException();
     }
 
+    const releaseTagPattern = plugin.settingsSchema ? await this.getPluginGithubReleaseTagPattern(pluginName) : null;
```

Comment:
does this need to default to `null` anyway if the function itself returns null in 'else' cases?